### PR TITLE
DOCS Bring CWP FTS docs inline with new official FTS docs

### LINKED ISSUE #126: Consolidate CWP docs with new fulltextsearch rewrite
# Overview

Andrew has done an awesome job in consolidating and updating the generic fulltextsearch documents for SS 4.x via https://github.com/silverstripe/silverstripe-fulltextsearch/pull/215. The aim for this work was for CWP developers to have a better pathway to both getting started with search implementations, and doing more sophisticated stuff. In that respect, the work has missed the mark since it hasn't addressed any docs improvements in the main [Features of the default codebase > Solr search](https://www.cwp.govt.nz/developer-docs/en/2/features/solr_search/) CWP docs.

# Acceptance Criteria

 * As a CWP developer, I get an overview of how CWP's search works when starting out with cwp.govt.nz docs
 * As a CWP developer, I have an obvious pathway into deeper module docs hosted on https://github.com/silverstripe/silverstripe-fulltextsearch, without conflicting or significantly overlapping advice on cwp.govt.nz
 * As a CWP developer, I have a consistent place for developer configuration on the main module docs (not one-offs like [CWP's "Solr queries using date ranges"](https://www.cwp.govt.nz/developer-docs/en/2/how_tos/solr_date_ranges/) or [Solr highlighting search results](https://www.cwp.govt.nz/developer-docs/en/2/how_tos/solr_highlighting_search_results/))
 * As a community developer, I don't get confused by CWP-specific advice (already the case e.g. in the  [remote installation](https://github.com/silverstripe/silverstripe-fulltextsearch/blob/master/docs/en/02_setup.md#remote-installation) section)
 * Any consolidated/moved URLs redirect correctly to their new location

### PR DESCRIPTION
**Pull request by dhensby**
closes #126 

This updates the docs for FTS in CWP so they are more inline with the core docs and link to relevant places.

### CODE REVIEW

File: docs/en/02_Features/01_Solr_search/01_Running_solr_locally.md [hunk 1]
```diff
@@ -12,31 +12,45 @@ When selecting versions, make sure you use the version supported by CWP: `4.*@de
...
+You can execute the quick start script by running `vendor/bin/fulltextsearch_quickstart` in your project root.
+
+## Differences on CWP
 
 Please read through the [Limitations and Acceptable Use Policy](configuration#limitations)
 of running Solr in the CWP infrastructure. 
 
-Most importantly, CWP infrastructure enforces `solrconfig.xml` to ensure stability of the shared service.
-While you can customise this file on your local development environment, these customisations will automatically be removed by the CWP Solr server, and the file restored to a version very similar to the default supplied by *fulltextsearch* module.
+Most importantly, CWP infrastructure does no permit customisations to `solrconfig.xml`, this is to ensure stability of the shared service.
```
Comment (robbieaverill) [inline docs/en/02_Features/01_Solr_search/01_Running_solr_locally.md]: not*

File: docs/en/02_Features/01_Solr_search/01_Running_solr_locally.md [hunk 2]
```diff
@@ -12,31 +12,45 @@ When selecting versions, make sure you use the version supported by CWP: `4.*@de
 Older versions of *silverstripe-fulltextsearch* module used to bundle the Solr binary. You should check the version of this module before proceeding with this guide.
 </div>
 
-## Differences to CWP
+Alternatively you can use the quickstart installer that ships with the latest version of the fulltextsearch module. The 
+quickstart script will install a solr service on your local machine and set up your `_config.php` to allow you to get
```
Comment (robbieaverill) [inline docs/en/02_Features/01_Solr_search/01_Running_solr_locally.md]: Solr*

File: docs/en/02_Features/01_Solr_search/01_Running_solr_locally.md [hunk 3]
```diff
@@ -12,31 +12,45 @@ When selecting versions, make sure you use the version supported by CWP: `4.*@de
...
 
 ## Configuring
 
 If you have configured your project as described in the Configuration section of this guide, there is no need for any additional setup.
 
 ## Running
 
-To start the local server instance (if you are using the *silverstripe/fulltextsearch-localsolr* package), from your
-website root do:
+### With localsorl
```
Comment (robbieaverill) [inline docs/en/02_Features/01_Solr_search/01_Running_solr_locally.md]: localsolr*

**Comment by robbieaverill** [comment]
I made a related PR to update the APIs we use in cwp-search: https://github.com/silverstripe/cwp-search/pull/12

**Comment by chillu** [comment]
Good stuff. Sent a follow up request with https://github.com/silverstripe/cwp/pull/136.

**Pull request merged**